### PR TITLE
Widget messages but hovers also work now

### PR DESCRIPTION
**Pull request by Vtec234**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -58,52 +57,259 @@
             }
         },
         "lean4-infoview-api": {
-            "name": "@leanprover/infoview-api",
```
Comment (mhuisi) [inline package-lock.json:61]: This probably shouldn't be removed.

File: package-lock.json [hunk 2]
```diff
@@ -11,21 +11,20 @@
...
                 "lerna": "^8.1.2",
                 "lint-staged": "^15.2.2",
-                "prettier": "3.2.5",
+                "prettier": "^3.2.5",
                 "prettier-plugin-organize-imports": "^3.2.4",
                 "typescript": "^5.4.5"
             }
         },
         "lean4-infoview": {
-            "name": "@leanprover/infoview",
```
Comment (mhuisi) [inline package-lock.json:28]: This probably shouldn't be removed.

File: package-lock.json (hunk 1 above)
Comment (llllvvuu) [reply package-lock.json:61]: If I run `npm install` on your branch it adds them back. My setup:

```
; node --version                                                         
v20.12.2
; npm --version          
10.5.0
```

**Review by mhuisi — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (Vtec234) [reply package-lock.json:61]: Yeah, not sure why my NPM keeps removing these.

Comment (Vtec234) [reply package-lock.json]: It seems that `npm update` removes them, and `npm install` readds them.

**Pull request merged**